### DISCUSSION TITLE
How to use when Native/AOT is enabled?

### DISCUSSION THREAD

**Discussion by nonunknown777** [aaubry/YamlDotNet#885] [Q&A] [answered]
I found no material on how to serialize a class or create an YAML manually when the native publish is enabled! Can someone show some examples?

**Comment by EdwardCooke** [comment, marked as answer]
I thought I built documentation on how to use this but I can't find it. Here's the github issue we worked on to get it working and built. Right now you need to use my static serializer nuget package to add the required code generator.

#740 

I'll work on some documentation somewhere for this shortly. It was pretty trivial to get it working.

Until I get that up, here's the example project on how to use it, you just need to reference my nuget package and you should be good.

https://github.com/aaubry/YamlDotNet/tree/master/YamlDotNet.Core7AoTCompileTest
